interop/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "multidim-interop",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Multidimension Interop Test",
+  "type": "module",
+  "main": "index.js",
+  "author": "Glen De Cauwsemaecker <glen@littlebearlabs.io> / @marcopolo",
+  "license": "MIT",
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "build": "aegir build",
+    "test": "aegir test"
+  },
+  "dependencies": {
+    "@chainsafe/libp2p-noise": "^12.0.0",
+    "@chainsafe/libp2p-yamux": "^4.0.1",
+    "@libp2p/mplex": "^8.0.1",
+    "@libp2p/tcp": "^7.0.1",
+    "@libp2p/webrtc": "^2.0.2",
+    "@libp2p/websockets": "^6.0.1",
+    "@libp2p/webtransport": "^2.0.1",
+    "@multiformats/mafmt": "^12.1.2",
+    "@multiformats/multiaddr": "^12.1.3",
+    "libp2p": "../",
+    "node-fetch": "^3.3.0",
+    "playwright": "^1.31.2",
+    "redis": "4.5.1"
+  },
+  "browser": {
+    "@libp2p/tcp": false
+  },
+  "devDependencies": {
+    "aegir": "^38.1.0",
+    "standard": "^17.0.0",
+    "typescript": "^4.9.4"
+  },
+  "overrides": {
+    "playwright-core": "^1.31.2"
+  }
```

Comment:
If I recall correctly, we needed a newer browser for some test. This shouldn't be needed after https://github.com/hugomrdias/playwright-test/pull/528/files.

If we package.json supported comments it would have been easier to capture the rational here.